### DISCUSSION TITLE
AL KAHERA WALNESS 2

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#780] [Q&A]
### Channel Name

ALKAHERA WALNESS 2

### Channel ID

AlKaheraWalNas2.eg

### Website

https://www.youtube.com/watch?v=y4ws3LzEV0U&ab_channel=%D8%A7%D9%84%D9%82%D8%A7%D9%87%D8%B1%D8%A9%D9%88%D8%A7%D9%84%D9%86%D8%A7%D8%B3

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ghost** [comment]
You need to scrape the url as it expires every few days

`curl -s 'http://alkaherawalnas.com/?page_id=14' | grep "youtube.com/embed" | sed -n 2p | sed 's/\"/\n/g' | grep "youtube.com/embed"`

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=kGIQMgz-ZSY for Walness 1

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=y4ws3LzEV0U for Walness 2

Might change if they cut off streams.

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=y4ws3LzEV0U (offline)
<img width="685" alt="image" src="https://user-images.githubusercontent.com/7253922/232227153-3e19e3fa-565c-4535-9e67-fd25571da532.png">

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19753
